src/models/CommonModel.ts

```diff
@@ -29,7 +29,7 @@ export class CommonModel extends CommonSchema<CommonModel> {
   getFromSchema<K extends keyof Schema>(key: K) {
     const schema = this.originalSchema || {};
     if (typeof schema === 'boolean') return undefined;
-    return schema[`${key}`];
+    return schema[String(key)];
```

Comment:
Why not simply `schema[key]`?

Reply:
It is because of `Generic Object Injection` Sink warning in eslint.

Based on the following discussion: [#184](https://github.com/asyncapi/generator-model-sdk/pull/184#discussion_r632278908)

Reply:
Got it. Thanks!